dags/resources/stages/parse/table_definitions/sudoswap/LSSVMPairFactory_call_bondingCurveAllowed.json

```diff
@@ -0,0 +1,38 @@
+{
+    "parser": {
+        "type": "trace",
+        "contract_address": "0xb16c1342e617a5b6e4b631eb114483fdb289c0a4",
+        "abi": {
+            "inputs": [
+                {
+                    "internalType": "contract ICurve",
+                    "name": "",
+                    "type": "address"
+                }
+            ],
+            "name": "bondingCurveAllowed",
+            "outputs": [
+                {
+                    "internalType": "bool",
+                    "name": "",
+                    "type": "bool"
+                }
+            ],
+            "stateMutability": "view",
+            "type": "function"
+        },
+        "field_mapping": {}
+    },
+    "table": {
+        "dataset_name": "sudoswap",
+        "table_name": "LSSVMPairFactory_call_bondingCurveAllowed",
+        "table_description": "",
+        "schema": [
+            {
+                "name": "",
```

Comment:
Will this work? I know we use `unnamedField<N>` placeholders [elsewhere](https://github.com/nansen-ai/evmchain-etl-table-definitions/search?q=unnamed).